content/python/concepts/built-in-functions/terms/input/input.md

```diff
@@ -25,8 +25,9 @@ The **`input()`** built-in function prompts the user for data that is converted
 ```pseudo
 input(prompt_string)
 ```
+In Python 3, **`input()`** is used while **`raw_input()`** was used in Python 2.7 
```

Comment:
```suggestion

> **Note:** The `input()` function is the primary user input function for Python 3.x while the `raw_input()` function was used in Python 2.7, with support and updates discontinued since January 2020.
```